Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -396,11 +397,26 @@ function Invoke-CodeGen {
     Invoke-Task -name $MyInvocation.MyCommand.Name -task {
         $tool = (Join-Path $toolsPath 'EdFi.Ods.CodeGen')
         $repositoryRoot = (Get-RepositoryRoot $implementationRepo).Replace($implementationRepo, '')
+        
+        $extensionLocationPlugins = @()
 
-        if ($IncludePlugins) {
+        if (-not [string]::IsNullOrWhiteSpace($ExtensionPaths)) {                        
```

Comment:
This logic still looks wrong..
If Include Plugins -> load plugins 
If extensions paths -> include extensions for each path

Plugins are independent of extension paths and should be treated as such. 

How do we handle if an extension and the plugin are the same?

Reply:
yes ,  I won't add if already exist based on Project.Name for domainModelDefinitionsByPath
Enumerable<AssemblyData> Get() which also removes duplicate also when it's returns.

Reply:
Different usecase as below 
Invoke-CodeGen -Engine SQLServer  -IncludePlugins
Invoke-CodeGen -Engine SQLServer -IncludePlugins -ExtensionPaths C:\OSS-Workspace\Semalai_EndUser\,C:\OSS-Workspace\EdFi.Ods.Extensions.TPDM\
Invoke-CodeGen -Engine SQLServer -ExtensionPaths C:\OSS-Workspace\Semalai_EndUser\,C:\OSS-Workspace\EdFi.Ods.Extensions.TPDM\
Invoke-CodeGen -Engine SQLServer 
Invoke-CodeGen -Engine SQLServer -IncludePlugins -ExtensionPaths C:\OSS-Workspace\Ed-Fi-Extensions\Extensions